concent_api/core/tests/test_payments_bankster.py

```diff
@@ -356,27 +358,29 @@ def test_that_settle_overdue_acceptances_should_return_none_if_subtask_costs_whe
         """
         self.create_subtask_results_accepted_list(price=3000)
 
-        claim_against_requestor = self.call_settle_overdue_acceptances_with_mocked_sci_functions()
+        with self.assertRaises(BanksterNoUnsettledTasksError):
+            self.call_settle_overdue_acceptances_with_mocked_sci_functions()
 
-        self.assertIsNone(claim_against_requestor)
         self.assert_mocked_sci_functions_were_called()
 
-    def test_that_settle_overdue_acceptances_should_return_none_if_requestor_deposit_value_is_zero(self):
+    def test_that_settle_overdue_acceptances_should_raise_too_small_requestor_deposit_exception_when_requestor_has_insufficient_funds(self):
         """
         In this test we have following calculations:
 
-        TaskToCompute price:                                    3000 (does not matter in this case)
+        TaskToCompute price:                                    12000 (does not matter in this case)
```

Comment:
Why this changed? Text says that it should not matter.

Reply:
ok, I will refactor it to get better clarity ;)